src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -93,6 +99,63 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String classDatetime} into a {@code Class}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code dateTime} is invalid.
+     */
+    public static Class parseClass(String classDatetime) throws ParseException {
+        requireNonNull(classDatetime);
+        String trimmedClassDatetime = classDatetime.trim();
+        if (!Class.isValidClassString(trimmedClassDatetime)) {
+            throw new ParseException(Class.MESSAGE_CONSTRAINTS);
+        }
+
+        // the format has been validated in isValidClassString method
+        // ie yyyy-MM-dd 0000-2359
+        String datetimeStr = trimmedClassDatetime.substring(0, 10);
+        String startTimeStr = trimmedClassDatetime.substring(11, 15);
+        String endTimeStr = trimmedClassDatetime.substring(16);
```

Comment:
Use of some magic numbers here. Could be better if they are declared as final constants at the top.